diktat-cli/src/test/kotlin/com/saveourtool/diktat/smoke/DiktatSmokeTestBase.kt

```diff
@@ -378,6 +379,13 @@ abstract class DiktatSmokeTestBase {
         fixAndCompare(prepareOverriddenRulesConfig(), "SemicolonsExpected.kt", "SemicolonsTest.kt")
     }
 
+    @Test
+    @Tag("DiktatRuleSetProvider")
+    @Timeout(TEST_TIMEOUT_SECONDS, unit = SECONDS)
+    fun `should add newlines between interfaces`() {
+        fixAndCompare(prepareOverriddenRulesConfig(), "NewlinesAfterInterfacesExpected.kt", "NewlinesAfterInterfacesTest.kt")
+    }
+
```

Comment:
Why do we need this smoke test?

Reply:
Maybe add fix test instead of smoke test?

Reply:
Added fix test